crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -846,15 +846,15 @@ pub struct RunArgs {
 
     // clap does not have negation flags such as --daemon and --no-daemon
     // so we need to use a group to enforce that only one of them is set.
```

Comment:
Can drop this comment line as well since we're not doing these shenanigans anymore

Reply:
I did drop the second line, but we do still use the group to enforce that you can't use both `--daemon` and `--no-daemon` don't we?

Reply:
Oops, you're right.